cask-cli.el

```diff
@@ -365,6 +379,7 @@ Commands:
  (command "help [command]" cask-cli/help)
  (command "load-path" cask-cli/load-path)
  (command "exec-path" cask-cli/exec-path)
+ (command "eval [form]" cask-cli/eval)
```

Comment:
Thinking about it, the `form` argument is not optional, so it should be: `"eval form"`